spyder/plugins/maininterpreter/tests/test_confpage.py

```diff
@@ -8,11 +8,12 @@
 
 # Standard library imports
 import time
+from unittest.mock import Mock, MagicMock
```

Comment:
I think this import is no longer necessary. If that's the case, please remove it.